docs/tutorial_notebooks/tutorial2/Introduction_to_PyTorch.ipynb

```diff
@@ -1299,7 +1299,997 @@
+      "image/svg+xml": [
```

Comment:
What's the change in these figures?

Reply:
Nothing changed. It seems my editor automatically reformatted this part after saving the file.